packages/server/postgres/migrations/1709749330256_retroReflectionGroups2.ts

```diff
@@ -0,0 +1,87 @@
+import {Kysely, PostgresDialect} from 'kysely'
+import {r} from 'rethinkdb-ts'
+import connectRethinkDB from '../../database/connectRethinkDB'
+import getPg from '../getPg'
+
+export async function up() {
+  await connectRethinkDB()
+  const pg = new Kysely<any>({
+    dialect: new PostgresDialect({
+      pool: getPg()
+    })
+  })
+  try {
+    await r
+      .table('RetroReflectionGroup')
+      .indexCreate('updatedAtId', (row: any) => [row('updatedAt'), row('id')])
+      .run()
+    await r.table('RetroReflectionGroup').indexWait().run()
+  } catch {
+    // index already exists
+  }
+
+  const MAX_PG_PARAMS = 65545
+  const PG_COLS = [
+    'id',
+    'createdAt',
+    'updatedAt',
+    'isActive',
+    'meetingId',
+    'promptId',
+    'sortOrder',
+    'voterIds',
+    'smartTitle',
+    'title',
+    'summary'
+  ] as const
+  type RetroReflectionGroup = {
+    [K in (typeof PG_COLS)[number]]: any
+  }
+  const BATCH_SIZE = ~~(MAX_PG_PARAMS / PG_COLS.length)
```

Comment:
+1 Had to look it up and the docs https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_NO mention `Math.trunc` should be used instead.